types/async/index.d.ts

```diff
@@ -123,18 +123,28 @@ export const eachOf: typeof forEachOf;
 export const eachOfSeries: typeof forEachOf;
 export const eachOfLimit: typeof forEachOfLimit;
 export function map<T, R, E = Error>(arr: T[] | IterableIterator<T> | Dictionary<T>, iterator: AsyncResultIterator<T, R, E>, callback?: AsyncResultArrayCallback<R, E>): void;
-export const mapSeries: typeof map;
```

Comment:
These actually had the same signature, so why couldn't you just add an overload to `map` and keep using `typeof map`?

Reply:
I didn't know if would work, since map has 2 overloads now
So i thought the typeof wouldn't know which one to take, so i played it safe and duplicated the the type with both overloads.

Reply:
Yeah, it will work. Just give it a shot, since I think the two were meant to stay in sync.